File: modules/day-01-readable/python/starter.py
scores = [88, 91, 76, 95, 84]
attendance = [1, 1, 0, 1, 1]
bonus = [2, 2, 0, 3, 1]
n = ["ana", "ben", "cam", "dev", "eli"]

f = []
for i in range(len(scores)):
    f.append((scores[i] * 0.8) + (attendance[i] * 10) + bonus[i])

print("results")
for i in range(len(f)):
    if f[i] >= 90:
        print(n[i], "did great")
    elif f[i] >= 80:
        print(n[i], "ok")
    else:
        print(n[i], "needs help")

print(sum(f) / len(f))

File: modules/day-01-readable/r/starter.R
scores <- c(88, 91, 76, 95, 84)
attendance <- c(1, 1, 0, 1, 1)
bonus <- c(2, 2, 0, 3, 1)
n <- c("ana", "ben", "cam", "dev", "eli")

d <- data.frame(n, scores, attendance, bonus)
d$f <- (d$scores * 0.8) + (d$attendance * 10) + d$bonus

print("results")
for (i in 1:nrow(d)) {
  if (d$f[i] >= 90) {
    print(paste(d$n[i], "did great"))
  } else if (d$f[i] >= 80) {
    print(paste(d$n[i], "ok"))
  } else {
    print(paste(d$n[i], "needs help"))
  }
}

print(mean(d$f))
